pydantic_core/_pydantic_core.pyi

```diff
@@ -35,6 +35,10 @@ __all__ = (
 __version__: str
 build_profile: str
 
+class Some(Protocol):
+    @property
+    def value(self) -> Any: ...
```

Comment:
This avoids creating a new public type, in practice it means that the only usage of this is accessing `value` for the object returned by `get_default_value` if it is None. You can't create or otherwise use this `Some` type.